### PR TITLE
CHE-345: allow offline machine creation

### PR DESCRIPTION
**Pull request by garagatyi**
Do not fail docker machine creation if registry is not reachable but needed image exists locally

@skabashnyuk @vparfonov

### CODE REVIEW

File: plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/parser/DockerImageIdentifierParser.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+ * Parse docker image reference.
+ * <p>
+ * For example reference used in FROM instruction of Dockerfile.<br>
+ * This class doesn't validate all components as Docker do.<br>
+ * It was designed to extract base docker image reference from dockerfile.
+ *
+ * @author Alexander Garagatyi
+ */
+public class DockerImageIdentifierParser {
+
```
Comment (skabashnyuk) [inline plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/parser/DockerImageIdentifierParser.java:28]: private constructor?

Comment (garagatyi) [inline plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/parser/DockerImageIdentifierParser.java:28]: forgot, 10x

Comment (garagatyi) [inline plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/parser/DockerImageIdentifierParser.java:28]: done

**Comment by vparfonov** [comment]
ok

**Comment by skabashnyuk** [comment]
ok

**Pull request merged**